extensions/amp-analytics/0.1/events.js

```diff
@@ -306,6 +334,89 @@ export class EventTracker {
   add(unusedContext, unusedEventType, unusedConfig, unusedListener) {}
 }
 
+/**
+ * Tracks browser events as a pass-through.
+ */
+export class BrowserEventTracker extends EventTracker {
+  /**
+   * @param {!./analytics-root.AnalyticsRoot} root
+   */
+  constructor(root) {
+    super(root);
+
+    /** @private {?Observable<!Event>} */
+    this.observables_ = new Observable();
+
+    /** @private {?Object<BrowserEventType>} */
```

Comment:
Sorry, my bad for the incorrect type earlier. Type should be `{!Object<BrowserEventType, boolean>}`.